content/en/docs/v3.6/learning/api.md

```diff
@@ -1,14 +1,18 @@
 ---
-title: etcd3 API
+title: etcd API
 weight: 2625
-description: etcd3 API central design overview
+description: etcd API central design overview
 ---
 
-This document is meant to give an overview of the etcd3 API's central design. It is by no means all encompassing, but intended to focus on the basic ideas needed to understand etcd without the distraction of less common API calls. All etcd3 API's are defined in [gRPC services][grpc-service], which categorize remote procedure calls (RPCs) understood by the etcd server. A full listing of all etcd RPCs are documented in markdown in the [gRPC API listing][grpc-api].
+This document is meant to give an overview of the v3 etcd APIs central design.
+This should not be mistaken with etcd v2 API, deprecated in etcd v3.5.
+It is by no means all encompassing, but intended to focus on the basic ideas needed to understand etcd without the distraction of less common API calls.
+All etcd APIs are defined in [gRPC services][grpc-service], which categorize remote procedure calls (RPCs) understood by the etcd server.
+A full listing of all etcd RPCs are documented in markdown in the [gRPC API listing][grpc-api].
```

Comment:
Same in this document with the splitting of paragraphs to individual lines.

Reply:
ditto